src/err/mod.rs

```diff
@@ -982,6 +982,24 @@ impl PyErrArguments for PyDowncastErrorArguments {
     }
 }
 
+/// Python exceptions that can be converted to [`PyErr`].
+///
+/// This is used to implement [`From<Bound<'_, T>> for PyErr`].
+///
+/// Users should not need to implement this trait directly. It is implemented automatically in the
+/// [`crate::import_exception!`] and [`crate::create_exception!`] macros.
```

Comment:
Should we also mention `impl_native_exception` and `impl_windows_native_exception` here? Anything else?

Reply:
I don't think so, they're intended to be internal implementation details. (Do we need to move them?)